x-pack/plugins/cases/public/containers/use_get_cases_metrics.test.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { renderHook, act } from '@testing-library/react-hooks';
+import * as api from '../api';
+import { TestProviders } from '../common/mock';
+import { useGetCasesMetrics, UseGetCasesMetrics } from './use_get_cases_metrics';
+import { SECURITY_SOLUTION_OWNER } from '../../common/constants';
+
+jest.mock('../api');
+jest.mock('../common/lib/kibana');
+
+describe('useGetReporters', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.restoreAllMocks();
+  });
+
+  it('init', async () => {
+    const { result } = renderHook<string, UseGetCasesMetrics>(() => useGetCasesMetrics(), {
+      wrapper: ({ children }) => <TestProviders>{children}</TestProviders>,
+    });
+
+    await act(async () => {
+      expect(result.current).toEqual({
+        mttr: 0,
+        isLoading: true,
+        isError: false,
+        fetchCasesMetrics: result.current.fetchCasesMetrics,
+      });
+    });
+  });
+
+  it('calls getCasesMetrics api', async () => {
+    const spy = jest.spyOn(api, 'getCasesMetrics');
+    await act(async () => {
+      const { waitForNextUpdate } = renderHook<string, UseGetCasesMetrics>(
+        () => useGetCasesMetrics(),
+        {
+          wrapper: ({ children }) => <TestProviders>{children}</TestProviders>,
+        }
+      );
+
+      await waitForNextUpdate();
+      expect(spy).toBeCalledWith({
+        http: expect.anything(),
+        signal: expect.anything(),
+        query: {
+          features: ['mttr'],
+          owner: [SECURITY_SOLUTION_OWNER],
+        },
+      });
+    });
+  });
+
+  it('fetch statuses', async () => {
```

Comment:
Could you please add another test that you call `fetchCasesMetrics`?

Reply:
This is actually fetchCasesMetrics. The test name is incorrect.

Reply:
I mean to do `result.current.fetchCasesMetrics()` and test the behaviour.

Reply:
gotcha. I'll add that test.